cli/cmd/vuln_container_list_assessments.go

```diff
@@ -385,6 +390,18 @@ func (v *treeCtrVuln) ParseData(data []api.VulnerabilityContainer) {
 	}
 }
 
+// Replace updates an existing ctrVuln in the treeCtrVuln slice with a new ctrVuln
+func (v *treeCtrVuln) Replace(old ctrVuln, new ctrVuln) {
+	current := *v
```

Comment:
If you have don't really need to use the pointer of `treeCtrVuln`, maybe switch this function to use the actual value so you can access and modify it with out doing `current := *v` and `*v = current`